src/styles/solar-stellarterm/styles/_forms.base.scss

```diff
@@ -17,6 +17,13 @@ html input.is-disabled {
   background-color: $s-color-neutral8;
   cursor: not-allowed;
 }
+
+.deposit-currency .dropdown {
```

Comment:
This does not belong in this file. `solar-stellarterm` is a theme.

Instead:
1. Make a `_Deposit.scss`
2. Include it in `app.scss`
3. Rename this class to `DepositCurrency__dropdown`

Reply:
ok, since I've added a second commit on top of this, is it ok with you if I leave this as-is in this commit, and then make this css change in the second commit which has the _Deposit.scss? that will be a little easier to make the change